velox/common/serialization/Serializable.h

```diff
@@ -179,35 +189,37 @@ class ISerializable {
   template <
       class T,
       typename = std::enable_if_t<has_static_obj_create_type<T>::value>>
-  static createReturnType<T> deserialize(const folly::dynamic& obj) {
-    return T::create(obj);
+  static createReturnType<T> deserialize(
+      const folly::dynamic& obj,
+      void* context) {
+    return T::create(obj, context);
   }
 
   template <
       class T,
       typename =
           std::enable_if_t<std::is_integral_v<T> && !std::is_same_v<T, bool>>>
-  static T deserialize(const folly::dynamic& obj) {
+  static T deserialize(const folly::dynamic& obj, void* context) {
```

Comment:
nit: you need to comment out the parameter name if you're not using it or lint will yell at you `void* /* context */`